r/R/expression.R

```diff
@@ -29,8 +29,17 @@
   # stringr spellings of those
   "str_length" = "utf8_length",
   "str_to_lower" = "utf8_lower",
-  "str_to_upper" = "utf8_upper"
+  "str_to_upper" = "utf8_upper",
   # str_trim is defined in dplyr.R
+  "year" = "year",
+  "isoyear" = "iso_year",
+  "quarter" = "quarter",
+  "month" = "month",
+  "isoweek" = "iso_week",
+  "day" = "day",
+  "yday" = "day_of_year",
+  "hour" = "hour",
+  "minute" = "minute"
```

Comment:
comment that second and wday are defined in dplyr-functions.R?

Reply:
Done